pages/common/pcapfix.md

```diff
@@ -0,0 +1,24 @@
+# pcapfix
+
+> Prepair pcap and pcapng files.
+> More information: <https://f00l.de/pcapfix/>.
+
+- Repair a pcap/pcapng file::
+
+`pcapfix {{test.pcap}}`
+
+- Repair entire pcap/pcapng file:
```

Comment:
```suggestion
- Repair an entire `pcap` file:
```
From the project's README: "This option is not necessary for pcapng files because the whole file is arranged
in blocks that are 'unlimited' by default."